crates/motionfile/src/motion_interpolator.rs

```diff
@@ -12,19 +13,20 @@ use types::ConditionInput;
 
 #[derive(Debug, Default, Serialize, Deserialize)]
 pub struct ConditionedSpline<T> {
-    pub entry_condition: Option<ConditionType>,
+    pub entry_condition: Option<DiscreteConditionType>,
+    pub enable: Option<Vec<ContinuousConditionType>>,
     pub spline: TimedSpline<T>,
-    pub exit_condition: Option<ConditionType>,
+    pub exit_condition: Option<DiscreteConditionType>,
 }
```

Comment:
Why can there be multiple `enable` conditions but only a single `entry_` or `exit_` condition?

Also why is it an `Option<Vec<...>>`? Why is `None` not just an empty vector? Or does `Option` allow you to omit the key in the JSON file? Then just use vectors and annotate it as serde-skip-field?

Reply:
`None` allows to omit the key in the json. I don't understand what you mean by `serde(skip)`. If I used that, I wouldnt be able to (de-)serialize that attribute anymore?

Reply:
About `Vec`: I thought it wouldn`t be too hard to implement this for enhanced functionality. However, you are right that it isn't used at the moment. Do you think I should drop `Vec` support?